app/src/main/res/values/strings.xml

```diff
@@ -76,22 +76,22 @@
     <!-- Dialogs -->
     <string name="button_dismiss">Dismiss</string>
     <string name="guide">Guide</string>
-    <string name="hold_on">Hold on!</string>
-    <string name="magisk_vanced">Looks like you\'re using the Magisk/TWRP version of Vanced, which is discontinued and can not be updated using this app. Please remove it first by removing the magisk module/using TWRP Vanced uninstaller.</string>
+    <string name="hold_on">Stop!</string>
+    <string name="magisk_vanced">You are using the Magisk/TWRP version of Vanced, which is discontinued and cannot be updated using this app. Please remove it by removing the magisk module/using TWRP Vanced uninstaller.</string>
     <string name="miui_one_title">MIUI detected!</string>
-    <string name="miui_one">Hey! it looks like you\'re a MIUI user. in order to properly use this application, you will have to disable MIUI optimisations in the developer settings. (You can safely ignore this warning if you are on a 20.2.20 or later ROM based on xiaomi.eu)</string>
+    <string name="miui_one">In order to properly install Vanced, you MUST disable MIUI Optimisations in the developer settings. (You can ignore this warning if you are on a 20.2.20 or later xiaomi.eu based ROM)</string>
     <string name="error">Error</string>
-    <string name="security_context">Please make sure, that you downloaded the app from vanced.app, the Vanced Discord server or the Vanced GitHub</string>
+    <string name="security_context">Make sure that you downloaded the app from vanced.app, the Vanced Discord server or the Vanced GitHub</string>
     <string name="success">Success!</string>
-    <string name="vanced_installed">Vanced has successfully been installed! Launch now?</string>
-    <string name="launch">Launch</string>
+    <string name="vanced_installed">Vanced has successfully been installed! Open now?</string>
+    <string name="launch">Open</string>
```

Comment:
There's a reason why we went with Launch instead of Open

Reply:
Open is shorter.